src/locales/fr.json

```
@@ -968,6 +968,12 @@
     "sortDateAsc": "Les moins fréquents en premier",
     "sortDateDesc": "Les plus fréquents en premier",
     "addingTagToTransaction": "Ajout du tag |||| Ajout des tags",
+    "addModal": {
+      "actions": {
+        "cancel": "Annuler",
+        "submit": "Ok"
```

```suggestion
        "submit": "Ajouter"
```
? ou alors seulement pour la surcharge ?
De même en anglais